src/lib/helpers/flag.ts

```diff
@@ -52,3 +52,41 @@ export const jsonStr = Flags.custom<AnyObj>({
     }
   },
 });
+
+/*
+ * Takes a flag input that's supposed to be a string containing multiple JSON string
+   and validates it.
+ */
+export const commaSeparatedJsonStr = Flags.custom<string[]>({
```

Comment:
This is something we need to decide @thomaswhyyou .

My idea was for the users to set the `--recipients` as a string containing multiple JSON strings:
```
workflow run new-comment --recipients='{"id": "franco1"}, {"id": "NON-EXISTING", "foo": "var"}, {"id": "foo", "collection": "projects"}' --data='{"x": "lele"}'
```

Another alternative is to receive this as a list of JSON's, which prevent us from doing the logic from above and maybe is better regarding user experience.

Thoughts?